src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
```

Comment:
```suggestion
    public Color getRandomColor() {
```
Don't use static methods in your solution (from common mistakes)